supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -2,13 +2,46 @@
 [[specific-doc-types]]
 = Specific documentation types
 
+[[prerequisites]]
+== Prerequisites
+
+When writing prerequisites, be as clear and concise as possible. You can use the passive voice, _if necessary_, to achieve that end.
+
+Write prerequisites as checks that are true or that the user must have completed before they begin a procedure. They can be actions that the user, another person, or piece of technology has completed. Prerequisites can also include items that the user must have ready before beginning the procedure.
+
+* The passive voice might be appropriate for a prerequisite that is not completed by the current user. For example, having a configuration enabled by a system admin.
+
+* Avoid using imperative formations.
+
+* Use parallel language when writing prerequisites, but it is not a requirement if you can achieve greater clarity without parallel language.
+
+.Examples of prerequisites
```

Comment:
Hi @stoobie 
Thanks for persisting with this one!
Maybe we don't need to add the detail about whether it's active or passive in each example? (seems a bit overkill) Probably only need to explain the first one because the passive is an outlier/exception.
And could we switch some out to include:
- a grocery list item. I use and see that a lot and think it's one of the more usual types of prereqs so should be included
- a present perfect item. This past tense is needed sometimes

 So I'm suggesting something like:

* JDK 11 or later is installed. (passive voice: the agent is unknown or unimportant.)

* A running Kafka instance in {product}.

* You are logged in to the Administration Portal.

* You have run a validation.

wdyt?

Reply:
I don't think this bullet is what we intended when we discussed this in our meetings. 
`*Use parallel language when writing prerequisites, but it is not a requirement if you can achieve greater clarity without parallel language.`

How about this instead?
`*Use parallel language when writing prerequisites. For example, if one bullet is a complete sentence, the other bullets should match. But one bullet could be passive voice and another active voice.`

I don't think parallelism is optional. It's just grammar.